pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_suite_test.go

```diff
@@ -0,0 +1,19 @@
+package nodenetworkconfigurationpolicy
+
+import (
+	"os"
+
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestUnit(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "NodeNetworkConfigurationPolicy controller Test Suite")
+}
+
+var _ = BeforeSuite(func() {
+	os.Setenv("NODE_NAME", "node01")
```

Comment:
Does this apply when we do it from init?

Reply:
I have to remove this damn, good catch.